lib/tests/integration-tests/search.integration.test.ts

```diff
@@ -14,13 +14,33 @@ function baseBuilder() {
     });
 };
 
-test('Relevance modifier without conditions', async() => {
+function baseProductCategoryBuilder() {
+    return new ProductCategorySearchBuilder({
+        language: 'da',
+        currency: 'DKK',
+        displayedAtLocation: 'integration test',
+        user: UserFactory.anonymous(),
+    });
+};
+
+test('ProductSearch: Relevance modifier without conditions', async() => {
 
-    const request: ProductSearchRequest = baseBuilder()
+    const request: ProductSearchRequest = baseProductBuilder()
         .relevanceModifiers(b => b.addProductDataRelevanceModifier('NoveltyBoostModifier', conditions => conditions, ValueSelectorFactory.dataDoubleSelector('NoveltyBoostModifier')))
         .build();
 
     const result = await searcher.searchProducts(request);
 
+    expect(result?.hits).toBeGreaterThan(0);
+});
+
+test('ProductCategorySearch: Relevance modifier without conditions', async() => {
+
+    const request: ProductCategorySearchRequest = baseProductCategoryBuilder()
+        .relevanceModifiers(b => b.addProductDataRelevanceModifier('NoveltyBoostModifier', conditions => conditions, ValueSelectorFactory.dataDoubleSelector('NoveltyBoostModifier')))
```

Comment:
Giver denne relevance modifier mening at teste her eller er det bare copy paste fra en anden test? :)

Reply:
Det var bare for at have den med tog ikke lige højde for indholdet :)